### ISSUE TITLE
`interactiveHandlers.json` includes non-interactive handlers

### ISSUE THREAD

**Issue by AdrianFahrbach** [vue-a11y/eslint-plugin-vuejs-accessibility#1381]
## The issue

The `interactiveHandlers.json` file should be a list of interactive handlers, but includes non-interactive handlers like `mouseenter` and `mousemove`.

This leads to issues with rules like [no-static-element-interactions](https://vue-a11y.github.io/eslint-plugin-vuejs-accessibility/rules/no-static-element-interactions.html), where an issue is thrown when an element only includes non-interactive like this:

```html
<div @mouseenter="emit('hover')"></div>
```

AFAIK W3C doesn't provide an official list of which events are interactive and which aren't, but the [eslint-plugin-jsx-a11y](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y) has the following events set as default in their [jsx-a11y/no-noninteractive-element-interactions config](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/a7d1a12a6198d546c4a06477b385b4fde03b762e/src/index.js#L119-L138):

```ts
'onClick',
'onError',
'onLoad',
'onMouseDown',
'onMouseUp',
'onKeyPress',
'onKeyDown',
'onKeyUp',
```

## Proposed solution

1. Adjust the `interactiveHandlers.json` to only interactive events. This is my preferred solution, but there is no clear definition of interactive events and this could affect other rules as well. Aligning with `eslint-plugin-jsx-a11y`would probably still be a good idea though.
2. `jsx-a11y/no-noninteractive-element-interactions` has an option to pass custom handlers. Adding options like that to rules using the interactive events would be a possible solution as well.
